x/examples/outline-connectivity-app/app_mobile/vite.config.ts

```diff
@@ -17,5 +17,8 @@ import { defineConfig } from "vite";
 export default defineConfig({
   build: {
     outDir: './output/frontend',
+  },
+  server: {
```

Comment:
How is this port used? Do you need to hardcode the port? What if there's a service there, will the app not work?

Reply:
As per above, bind it to localhost here too. The default is probably all IPs.